code/modules/surgery/bodyparts/robot_bodyparts.dm

```diff
@@ -265,16 +265,16 @@
 /obj/item/bodypart/head/robot/examine(mob/user)
 	. = ..()
 	if(!flash1 && !flash2)
-		. += "<span class='info'>It has two empty eye sockets for <b>flashes</b>.</span>"
+		. += "<span class='info'>It has two empty eye sockets for <b>cyborg eyes</b>.</span>"
 	else
 		var/single_flash = FALSE
 		if(!flash1 || !flash2)
 			single_flash = TRUE
-			. += "One of its eye sockets is currently occupied by a flash.\n"+\
-			"<span class='info'>It has an empty eye socket for another <b>flash</b>.</span>"
+			. += "One of its eye sockets is currently occupied by a [flash1 ? "[flash1.name]":"[flash2.name]"].\n"+\
+			"<span class='info'>It has an empty eye socket for another <b>cyborg eye</b>.</span>"
 		else
-			. += "It has two eye sockets occupied by flashes."
-		. += "<span class='notice'>You can remove the seated flash[single_flash ? "":"es"] with a <b>crowbar</b>.</span>"
+			. += "It has two eye sockets occupied by a [flash1.name] and [flash2.name]."
+		. += "<span class='notice'>You can remove the seated eye[single_flash ? "":"s"] with a <b>crowbar</b>.</span>"
```

Comment:
aren't there macros for this in the language? like `\plural` or something?

Reply:
Not one that's applicable here. (it's `\s`)